cmake/fftw_on_windows.cmake

```diff
@@ -39,13 +39,28 @@ CheckExitCodeAndExitIfError("tar")
 
 get_filename_component(_vs_bin_path "${CMAKE_LINKER}" DIRECTORY)
 
-execute_process(
-    COMMAND "${_vs_bin_path}/lib.exe" ${LIBEXE_64} /def:${CMAKE_BINARY_DIR}/fftw/libfftw3-3.def /out:${CMAKE_BINARY_DIR}/fftw/libfftw3-3.lib
-    WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/fftw"
-    OUTPUT_VARIABLE OutVar
-    ERROR_VARIABLE ErrVar
-    RESULT_VARIABLE ExitCode)
-CheckExitCodeAndExitIfError("lib.exe: ${OutVar} ${ErrVar}")
+# use dlltool in case of system is unix and target system is windows
+# not sure about binary name, maybe it can be differ on some distros. 
```

Comment:
Please remove this comment. Add a user visible message instead. You can check `ExitCode` like this:
```
if (NOT ExitCode)
 message(FATAL_ERROR "Your cross compiler is not installed or called different then i686-w64-mingw32-dlltool")
end()
```